core/src/main/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParser.java

```diff
@@ -40,6 +40,12 @@ public class ClusterConfigParser {
   private static final String PROPERTY_FORMAT = "%s=\"%s\"%n";
   private static final String[] SECTIONS = new String[] {"manager", "monitor", "gc", "tserver"};
 
+  private static final Set<String> VALID_CONFIG_KEYS = Set.of("manager", "monitor", "gc", "tserver",
+      "tservers_per_host", "sservers_per_host", "compaction.coordinator");
+
+  private static final String[] VALID_CONFIG_PREFIXES =
+      new String[] {"compaction.compactor.", "sserver."};
```

Comment:
If you make these into a Predicate, this makes the code that streams it look a lot cleaner. The Predicate itself can be simplified, too, if you stream the prefixes, rather than for loop over it.

Here's the Predicate (and making the array into a Set):

```suggestion
  private static final Set<String> VALID_CONFIG_PREFIXES =
      Set.of("compaction.compactor.", "sserver.");

  private static final Predicate<String> VALID_CONFIG_SECTIONS =
      section -> VALID_CONFIG_KEYS.contains(section)
          || VALID_CONFIG_PREFIXES.stream().anyMatch(section::startsWith);
```

Note: this requires `import java.util.function.Predicate;`